docs/workflow/testing/libraries/testing.md

```diff
@@ -79,13 +79,25 @@ cd src\libraries\System.Collections.Immutable\tests
 dotnet build /t:Test
 ```
 
-It is possible to pass parameters to the underlying xunit runner via the `XUnitOptions` parameter, e.g.:
+This one command builds the library under test, then the test library, then runs the tests in the test library.
```

Comment:
This isn't generally the case.  It's true for S.C.Immutable because the test project has a project reference to the src project, but for example the System.Net.Http test project does not have a project reference to the src project and does not cause the library under test to be built.